src/Symfony/Component/Translation/Translator.php

```diff
@@ -395,7 +395,7 @@ private function getCatalogueCachePath($locale)
         return $this->cacheDir.'/catalogue.'.$locale.'.'.strtr(substr(base64_encode(hash('sha256', serialize($this->fallbackLocales), true)), 0, 7), '/', '_').'.php';
     }
 
-    private function doLoadCatalogue($locale): void
+    protected function doLoadCatalogue($locale): void
```

Comment:
What about marking this method as `@internal` now that it is protected?

Reply:
Makes sense. I'll change that.